core/crypto/client_tcert_pool.go

```diff
@@ -23,7 +23,9 @@ type tCertPool interface {
 
 	Stop() error
 
-	GetNextTCert() (tCert, error)
+	GetNextTCerts(nCerts int) ([]tCert, error)
```

Comment:
Shall we avoid to expose internal interfaces? The interface *tCert* is anyway unusable outside the crypto package by the golang rules.
The equivalent public interface is CertificateHandler.